src/wallet/feebumper.cpp

```diff
@@ -241,12 +241,13 @@ bool CFeeBumper::commit(CWallet *pWallet)
     if (!vErrors.empty() || currentResult != BumpFeeResult::OK) {
         return false;
     }
-    if (txid.IsNull() || !pWallet->mapWallet.count(txid)) {
+    auto it = pWallet->mapWallet.find(txid);
+    if (txid.IsNull() || it == pWallet->mapWallet.end()) {
```

Comment:
@ryanofsky I guess it's to prevent a unnecessary `count()` in the old code. What about:
```cpp
auto it = txid.IsNull() ? pWallet->mapWallet.end() : pWallet->mapWallet.find(txid);
if (it == pWallet->mapWallet.end()) {
```